src/landscape/components/LandscapeView.js

```diff
@@ -116,19 +102,44 @@ const LandscapeView = () => {
   return (
     <PageContainer>
       <PageHeader header={landscape.name} />
-      <Typography
-        variant="caption"
-        display="block"
-        sx={{
-          marginBottom: theme.spacing(3),
-          marginTop: theme.spacing(2),
-        }}
-      >
-        {landscape.location}
-      </Typography>
+      <Restricted permission="landscape.change" resource={landscape}>
+        <Alert severity="info" sx={{ marginBottom: 2 }}>
+          <Trans i18nKey="landscape.view_manager_help">
+            {{ name: t('user.full_name', { user }), landscape: landscape.name }}
+            <Link href="TODO">link</Link>.
```

Comment:
issue: link is still TODO

Reply:
@paulschreiber The help links are still not ready that is why this one says TODO and the others are empty

Reply:
issue: since the links will be localized, put a `t()` call in here and put the TBD link the the JSON file.